en/micro-integrator/docs/references/connectors/salesforce-soap-connector/sf-soap-connector-config.md

```diff
@@ -0,0 +1,53 @@
+# Salesforce SOAP Connector Configuration
+
+The Salesforce SOAP connector allows you to access the [Salesforce SOAP API](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_quickstart_intro.htm?search_text=SOAP%20API%20Developer%20Guide) through WSO2 EI. 
+
+## Setting up the Salesforce account
+
+1. To work with the Salesforce SOAP connector, you need to have a Salesforce account. If you do not have a Salesforce account, go to [https://developer.salesforce.com/signup](https://developer.salesforce.com/signup) and create a Salesforce developer account.
+
+2. After creating a Salesforce account you will get a [Salesforce security token](https://help.salesforce.com/articleView?id=user_security_token.htm&type=5). 
+
+3. To configure the Salesforce SOAP Connector you need to save and keep the **username**, **password**, and **security token** of your Salesforce account.
+
+## Importing the Salesforce Certificate
+
+To use the Salesforce connector, add the `<salesforce.init>` element to your configuration before carrying out any other Salesforce operations.
+
+Before you start configuring the connector, import the **Salesforce certificate** to your WSO2 Enterprise Integrator (EI) **client keystore**.
+
+Follow the steps below to import the Salesforce certificate into the EI client keystore:
+
+1. To view the certificate, log in to your Salesforce account in your browser.
+2. Search the **Certificate and Key Management** in the search box.
+
+   <img src="../../../../assets/img/connectors/salesforcesoap-certificste-and-key-management.png" title="salesforcesoap-certificste-and-key-management" width="600" alt="salesforcesoap-certificste-and-key-management"/>  
+    
+3. Export the certificate to the file system.
+4. Import the certificate to the EI client keystore using either the following command or the [EI Management Console](https://docs.wso2.com/display/ESB490/Import+Certificates+for+Keystores):
+
+    ```
+    keytool -importcert -file <certificate file> -keystore <EI>/repository/resources/security/client-truststore.jks -alias "Salesforce"
+    ```
+
+5. Restart the server and deploy the following Salesforce configuration:
+
+    ```
+    <salesforce.init>
+        <username>MyUsername</username>
+        <password>MyPassword</password>
+        <loginUrl>https://login.salesforce.com/services/Soap/u/42.0</loginUrl>
+        <blocking>false</blocking>
+    </salesforce.init>
+    ```
+
+> **Note**: Secure Vault is supported for [encrypting passwords](https://docs.wso2.com/display/ADMIN44x/Encrypting+Passwords+with+Cipher+Tool). See, Working with Passwords on integrating and using Secure Vault.
```

Comment:
Do we have this in the latest docs? Is it https://ei.docs.wso2.com/en/7.1.0/micro-integrator/setup/security/single_key_encryption/

Reply:
yes.Updated.https://ei.docs.wso2.com/en/7.1.0/micro-integrator/setup/security/encrypting_plain_text/